api/v1/placementbinding_types.go

```diff
@@ -40,6 +40,13 @@ type PlacementSubject struct {
 // PlacementBindingStatus defines the observed state of PlacementBinding
 type PlacementBindingStatus struct{}
 
+// RemediationActionOverride defines the overrides of the policy RemediationAction
+type RemediationActionOverride struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=Enforce;enforce
```

Comment:
I started wondering if the opposite case exists. Could someone want to keep the default action as enforce for a set of clusters and override to inform for another set or a subset of the parent placement?

Reply:
I was wondering the same thing. Could the `RemediationAction` field from the Policy struct be referenced here instead of redefining it here?
https://github.com/open-cluster-management-io/governance-policy-propagator/blob/0370f385cb946b4ebf01ad9b320b13ab5d1eff5c/api/v1/policy_types.go#L13-L14

Reply:
I think the logic was simplified by only being able to override in one direction. If we allow an override to `inform`, then what happens if I have two Bindings, one overriding to `enforce`, one overriding to `inform`, and they *both* target the same cluster?

It's not in general a bad idea, but it complicates things.

Reply:
Yes, definitely. We can avoid this for now and expand the enum later which should be a backward compatible API change, when we have a concrete use case for the opposite override. However, we will have to deal with this kind of conflict as soon as we extend the overrides to other non binary fields, such as evaluation interval or dependency list. Each override may need different conflict resolution. A common one could be reporting as an error if we decide to be super strict on multiple bindings with overlapping overrides.